core/src/messages/agent.rs

```diff
@@ -142,11 +140,11 @@ impl BackendStatsMessage {
         //disk
```

Comment:
Is this meant to be a todo? If so, turn into a proper todo comment, if not, rm it